checkov/arm/graph_builder/local_graph.py

```diff
@@ -156,6 +160,25 @@ def _create_vars_and_parameters_edges(self) -> None:
                         var_name = match[1]
                         self._create_edge(var_name, origin_vertex_index, attr_key)
 
+    def _create_edge(self, element_name: str, origin_vertex_index: int, label: str) -> None:
+        dest_vertex_index = self.vertices_by_name.get(element_name)
+        if origin_vertex_index == dest_vertex_index or dest_vertex_index is None:
+            return
+        edge = Edge(origin_vertex_index, dest_vertex_index, label)
+        self.edges.append(edge)
+        self.out_edges[origin_vertex_index].append(edge)
+        self.in_edges[dest_vertex_index].append(edge)
+
+    def _create_implicit_edges(self, origin_vertex_index: int, resource_name: str, d: dict[str, Any]) -> None:
+        for _, value in d.items():
```

Comment:
why not just go over `d.values()` if you don't use the key

Reply:
Done